android/cli/hooks/aar-transform.js

```diff
@@ -481,3 +446,96 @@ class SimpleFileCache {
 		fs.writeFileSync(this.cachePathAndFilename, dataToWrite);
 	}
 }
+
+/**
+ * Hyperloop versions below 2.2.0 have their own AAR handling which we need to
+ * disable by hooking into the affected hooks and reverting the changes Hyperloop
+ * made.
+ *
+ * @param {Object} cli CLI instance
+ * @param {Object} builder Builder instance
+ * @param {Object} logger Logger instance
+ */
+function registerHyperloopCompatibilityFixes(cli, builder, logger) {
+	var hyperloopModule = null;
+	builder.nativeLibModules.forEach(function (module) {
```

Comment:
This should be a `builder.nativeLibModules.some()` call and when you find the hyperloop module, `return true;` to break the looping.